docs/includes/dev-kit-stop-debugging.md

```diff
@@ -0,0 +1,7 @@
+## Stop testing your Office Add-in
+
+Once you are finished testing and debugging the add-in, close the add-in by following these steps.
```

Comment:
This is important enough that we should emphasize it. There can be buggy behavior the next time they create an add-in if they don't shut down properly.

```suggestion
Once you are finished testing and debugging the add-in, *always* close the add-in by following these steps. (Closing the Office application or web server window doesn't reliably deregister the add-in.)
```